deno.json

```diff
@@ -2,7 +2,7 @@
   "tasks": {
     "build": "deno run --allow-read --allow-write --allow-net --allow-run --allow-env ./style/patch.ts && deno fmt",
     "check:types": "deno check **/*.ts",
-    "coverage": "deno test --allow-read --coverage=cov_profile",
+    "coverage": "rm -rf cov_profile && deno test --allow-read --coverage=cov_profile",
```

Comment:
This made testing things easier.  I don't think this causes any problems either.